### DISCUSSION TITLE
About Lory sliders

### DISCUSSION THREAD

**Discussion by censonalvin** [GrapesJS/grapesjs#5266] [Q&A]
Is there any way to control how many slide it will be? i was browsing this [https://github.com/GrapesJS/components-lory](url) and cant find how to control how many slide it will be and also cant find how to change the category of it. thanks in advance

**Comment by artf** [comment]
https://github.com/GrapesJS/components-lory/issues/13#issuecomment-1491881898

**Reply by censonalvin** [reply]
what slider/carousel can you suggest for v0.19 (grapesjs) ?

**Comment by censonalvin** [comment]
what should i do to add also on the canvas ? because i was successfully add a content/value to the slideEls of Lory slider but it doesn't add on or not showing on the canvas

here is what i added on the script plugin (dunno if it is the right way and i only added the trait)

>{
    type: 'button',
    name: 'add-slide',
    text: 'Add Slide',
    full: true, // Full width button
    command: function () {
        const addSlide = `\n      <div class="gjs-lory-slide test4"></div>`
        t.slideEls += addSlide;
        alert("slide els: " + addSlide + '\n' + t.slideEls);
    } 
}
